control/plugin/cpolicy/tree.go

```diff
@@ -140,28 +136,23 @@ func (c *ConfigPolicy) Get(ns []string) *ConfigPolicyNode {
 		return &t
 	default:
 		return t.(*ConfigPolicyNode)
-
 	}
 }
 
-func (c *ConfigPolicy) GetAll() map[string]*ConfigPolicyNode {
-	// Automatically freeze on first Get
-	if !c.config.Frozen() {
-		c.config.Freeze()
-	}
+type keyNode struct {
+	Key  []string
+	Node *ConfigPolicyNode
```

Comment:
I made this change. Fairly small diff but I made it as a separate commit so it'd be easy to verify (or notice if I missed something obvious).